src/Connector.SqlServer/Features/IBuildStoreDataFeature.cs

```diff
@@ -13,6 +15,10 @@ public interface IBuildStoreDataFeature
             string containerName,
             IDictionary<string, object> data,
             IList<string> keys,
+            StreamMode mode,
+            string correlationId,
+            DateTimeOffset timestamp,
+            VersionChangeType changeType,
```

Comment:
Breaking changes - We're introducing new interfaces to support the new types - if the old interface supports the default implementation, then introduce a new interface to support the new mode.

Reply:
Arent we adding mess + pain just to support Semler overrides which will take 5 mins to upgrade to 3.2.5?

Reply:
There are other connectors already created that are not for semler.  Every customer will be broken by this change and forced to update their connectors as well as server.

Non-of that is a 'five' minute upgrade - you have to consider the impact on production rollout and testing.

By keeping things working in a non-breaking manner users can upgrade relevant parts when ready.